libs/shared/ui/src/lib/styles/components/select-multiple.scss

```diff
@@ -15,9 +15,14 @@
 .input--select-multiple__value-container {
   padding: 0 !important;
   cursor: pointer;
+  display: flex !important;
```

Comment:
To example here, I understand why you use the !important we have default class by the react-select library